src/Scripting/CSharpTest/InteractiveSessionReferencesTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+using static Microsoft.CodeAnalysis.Scripting.TestCompilationFactory;
+
+namespace Microsoft.CodeAnalysis.CSharp.Scripting.Test
+{
+    public class InteractiveSessionReferencesTests : TestBase
+    {
+        /// <summary>
+        /// Test adding a reference to NetStandard 2.0 library.
+        /// Validates that we resolve all references correctly in the first and the subsequent submissions.
+        /// </summary>
+        [Fact]
+        [WorkItem(345, "https://github.com/dotnet/try/issues/345")]
+        public async Task LibraryReference_NetStandard20()
+        {
+            var libSource = @"
+public class C
+{
+    public readonly int X = 1;
+}
+";
+            var libImage = CreateCSharpCompilation(libSource, TargetFrameworkUtil.NetStandard20References, "lib").EmitToArray();
+
+            var dir = Temp.CreateDirectory();
```

Comment:
>dir [](start = 16, length = 3)

Do we need to dispose this object in any way?

Reply:
Directories and files created off of `Temp` are automatically disposed by the base class.